template/5-mvp.md

```diff
@@ -2,68 +2,92 @@
 
-- "As a social media enthusiast, I want to discover posts around me so that I can explore new places and share my experiences."
-- "As a user, I want to create posts with a title, description, and location to share interesting places I visit."
-User Stories for Travelers:
-- "As a traveler, I want to find unique local experiences by viewing posts from different cities."
-User Stories for Local Explorers:
-- "As a local explorer, I want to discover hidden gems in my city by exploring posts from other residents."
+#### User Stories for Local Explorers:
+1. "As a local explorer, I want to discover and hidden gems in my city by exploring posts from other residents."
+2. "As a local resident, I want to create local challenges that encourage people to explore lesser-known areas of my city."
 
-*Identify and prioritise the key features required.*
-*Justify the importance of each feature.*
-
-- User Authentication: Necessary for security and personalized user experiences.
-- Post Creation: Central to the app's purpose of sharing and discovering locations.
-- Location-Based Feed: Key to the unique value proposition of Proxima, encouraging physical exploration.
-- Commenting System: Enhances user interaction and engagement.
-- Offline Mode: Ensures usability even without internet access, improving user retention.
-- Voting System: Helps surface the best content and enhances user experience.
-- Posts and comments management: Allows users to track and manage their activity.
-- Centauri Points: Adds a competitive and rewarding element, increasing user engagement.
-- Challenges: Drives user activity and exploration, fostering a dynamic user community.
-- Map Integration: Allows users to view posts on a map and navigate to specific locations.
+#### Key Features:
+- **User Authentication:** Necessary for security and personalized user experiences.
+- **Post Creation and Location-Based Feed:** The core functionality of sharing and discovering places.
+- **Interaction Tools (Comments and Voting):** Enhances user interaction and engagement.
+- **Centauri Points and Challenges:** Motivate continued app usage by creating challenges with rewards in form of points
+- **Ranking System:** Encourages competition and engagement by displaying user rankings based on Centauri points.
+- **Map:** Allows users to navigate to post locations and to visualize nearby activities.
+- **Posts and comments management:** Allows users to track and manage their activity.
+- **Offline Mode:** Ensures usability even without internet access, improving user retention.
 
 
 ## Success Criteria
 
-*How will you evaluate the success of the MVP?*
-*Metrics include user penetration, quality / satisfaction.*
-*If applicable, progress in discussions with ecosystem partners / investors / customers.*
+#### Evaluation Metrics:
+- **User Adoption:** Measured by download numbers and active accounts.
+- **Engagement Metrics:** Frequency of app usage, interactions per user, and completion of challenges.
```

Comment:
I wonder if we could be a little more specific.
Maybe something like :

```suggestion
- **Engagement Metrics:** Daily active accounts, daily average interactions per user (creating post or comments, completing challenges).
```

Reply:
Thanks, 256b98d2731a8bc1284f42f90fe85cf61bc32b91